File: javascript/rna-transcription/rna-transcription.js
const dnaToRnaTable = {
  'G': 'C',
  'C': 'G',
  'T': 'A',
  'A': 'U'
}

export const toRna = (dna) => dna.toUpperCase().split('').map(code => dnaToRnaTable[code]).join('');

File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    dna_to_rna_table = {
            "G": "C",
            "C": "G",
            "T": "A",
            "A": "U"
            }

    return "".join(dna_to_rna_table[acid] for acid in dna_strand)
